CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ First bugfix release for version 2.1.0.
 - Certificate generation: Added a notice below the generate button ([#94](https://github.com/sovity/authority-portal/issues/94))
 - Removed "Control Center" from breadcrumb path to improve UX ([#54](https://github.com/sovity/authority-portal/issues/54))
 - Restricted user re-/deactivation to appropriate statuses ([#82](https://github.com/sovity/authority-portal/issues/82))
+- Add tooltip with info about participant roles ([#76](https://github.com/sovity/authority-portal/issues/76))
```

Comment:
This is in the wrong section of the changelog. You put it under 2.1.0 which is already released.

Reply:
oh thanks!